src/ui/backend/canvas/ViewBacking.js

```diff
@@ -25,12 +25,19 @@ import ..BaseBacking;
 import util.setProperty as setProperty;
 
 var _styleKeys = {};
+var IDENTITY_MATRIX = { a: 1, b: 0, c: 0, d: 1, tx: 0, ty: 0 };
```

Comment:
Minor optimization might be to move this constant matrix and the `sin` and `cos` vars into the constructor function to reduce the scope look-up by 1.